pkg/qemu-ga/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM alpine:3.5
+RUN apk add --update qemu-guest-agent
+CMD ["/usr/bin/qemu-ga"]
```

Comment:
could you also add a `LABEL org.mobyproject.config` similar to the one in `./pkg/sysctl/Dockerfile` as this makes the YAML files a lot simpler